app/Http/Controllers/InternetController.php

```diff
@@ -28,7 +28,8 @@ public function index()
     public function admin()
     {
         $activationDate = env('INTERNET_ACTIVATION_DATE'); //TODO: get date for current semester
-        return view('admin.internet.app', ['activation_date' => $activationDate]);
+
+        return view('admin.internet.app', ['activation_date' => $activationDate, 'users' => User::all()]);
```

Comment:
I might open a new issue for this. It is because the search-user function, and yes - it's unnecessary. 
Maybe the best solution would be if the search-user template requested the needed info with ajax.

Reply:
Okay, let's open an issue for that.